test-gui.sh

```diff
@@ -2,14 +2,17 @@
 
 set -euo pipefail
 
-source ./contrib/environment.sh
```

Comment:
The environment.sh script was setting env vars that don't work for OpenShift clusters.

Reply:
We have `oc-environment.sh`, that's what I use.